src/pages/Provider/Provider.layout.vue

```diff
@@ -48,9 +49,10 @@
                 Precio minimo
                 <q-input
                   type="number"
-                  suffix="$"
+                  prefix="S/."
```

Comment:
can you add a min value of, let say S/. 0.01

Reply:
Of course, will do!

Reply:
![lastrecord](https://user-images.githubusercontent.com/66505715/190185430-bbe6c512-bac1-42f3-b9ff-4b5c2656e5b5.gif)